packages/features/eventtypes/components/EventType.tsx

```diff
@@ -31,24 +29,19 @@ export type Host = {
 
 export type CustomInputParsed = typeof customInputSchema._output;
 
-const querySchema = z.object({
-  tabName: z
-    .enum([
-      "setup",
-      "availability",
-      "apps",
-      "limits",
-      "instant",
-      "recurring",
-      "team",
-      "advanced",
-      "workflows",
-      "webhooks",
-      "ai",
-    ])
-    .optional()
-    .default("setup"),
-});
+const tabs = [
+  "setup",
+  "availability",
+  "team",
+  "limits",
+  "advanced",
+  "instant",
+  "recurring",
+  "apps",
+  "workflows",
+  "webhooks",
+  "ai",
```

Comment:
don't we need to pass "payments" here too?

Reply:
payment is within apps

Reply:
for web at least, let's see what we do when we add payments to platform